examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor_postprocess.py

```diff
@@ -112,3 +117,52 @@
 ax0.legend(loc="upper left")
 fig0.savefig(f'./spike_and_bubble_evolution.png')
 plt.show()
+
```

Comment:
At line 81, you have to change the type of the data "phase" to "filtered_phase" to process data for filtered results